HelpSource/Tutorials/Getting-Started/04-Functions-and-Other-Functionality.schelp

```diff
@@ -13,20 +15,32 @@ Not too inspiring? Don't worry, we're just getting started, and this is just a s
 
 Before we get to doing that though, let's learn a little about Functions in general.
 
-A Function is just a reusable bit of code. You define a Function by enclosing code in 'curly brackets': { }. Here's an example:
+A Function is just a reusable bit of code. You define a Function by enclosing code in curly brackets: { }. Here's an example:
 
 code::
 f = { "Function evaluated".postln; };
```

Comment:
let's remove the semicolon in the function here: 

f = { "Function evaluated".postln };

Reply:
good catch, will do!